test/system/250-systemd.bats

```diff
@@ -295,6 +296,8 @@ LISTEN_FDNAMES=listen_fdnames" | sort)
     unit_file="contrib/systemd/system/${unit_name}"
     if [[ -e ${unit_file}.in ]]; then
         echo "# [Building & using $unit_name from source]" >&3
+        # Force regenerating unit file (existing one may have /usr/bin path)
+        rm -f $unit_file
```

Comment:
Oh, this was nasty. Basically, the unit file already existed, and had hardcoded `/usr/bin/podman`. Nuff said.